src/hdio.c

```diff
@@ -25,8 +25,7 @@ static int
 print_hdio_getgeo(struct tcb *const tcp, const kernel_ulong_t arg)
 {
 	if (entering(tcp)) {
-		tprint_arg_next();
-
+		tprints_arg_next_name("argp");
 		return 0;
 	}
```

Comment:
Likewise, I suggest moving this tprints_arg_next_name from the entering phase to the exiting.